01-basics/10-create-app/script.js

```diff
@@ -1 +1,20 @@
 import { defineComponent, createApp } from 'vue/dist/vue.esm-bundler.js'
+
+const App = defineComponent({
+  name: 'App',
+
+  setup() {
+    const date = new Date().toLocaleString('en', { dateStyle: 'long' })
+    return {
```

Comment:
It's preferred to use `navigator.language` instead of a fixed value. Then a user can see the format they've chosen in the web-browser settings.